js/bitfinex2.js

```diff
@@ -19,6 +19,27 @@ module.exports = class bitfinex2 extends bitfinex {
             'pro': false,
             // new metainfo interface
             'has': {
+                'signIn': false,
+                'setPositionMode': false,
+                'setMarginMode': false,
+                'setLeverage': false,
+                'reduceMargin': false,
+                'fetchWithdrawal': false,
+                'fetchTransfers': false,
+                'fetchPosition': false,
```

Comment:
Bitfinex v2 has `fetchPositions`: https://docs.bitfinex.com/reference#rest-auth-positions.
We should set those methods to `undefined` instead, if it is unknown whether the exchanges have them or not.
We should only set them to `false` if we are 100% sure that those methods are missing on the exchange side after checking their api. 

In general, I think we should not change these things in bulk, cause that will inevitably cause too many errors.

Reply:
I can leave the undefined methods, however it's nice to see the entirety of the ccxt api in one place

Reply:
i don't really think undefined should mean exists but not implemented yet, instead we should just use false or true to signify if the method is implemented or not. Cause having three values is less intuitive than two

Reply:
> i don't really think undefined should mean exists but not implemented yet

It does not have that meaning. The meaning of `undefined` is different. It means, we don't know if it's available or not. A value of `undefined` means we don't know and this has to be resolved. And the value of `false` means the exchange does not have it.

We should definitely make this PR conform with our own docs:

- https://github.com/ccxt/ccxt/wiki/Manual#exchange-metadata

> instead we should just use false or true to signify if the method is implemented or not. Cause having three values is less intuitive than two

No. Because the value of `undefined` helps the devs find out which of the methods were resolved and which are yet to be resolved.

Reply:
In general, a blind find-and-insert approach won't work – we have to do this process manually by reconciling with the exchanges' docs. For the future, let's not attempt bulk changes like these, it's very easy to miss details in this mode. We should be pedantic.